src/common/subscription/SubscriptionViewer.ts

```diff
@@ -142,7 +155,7 @@ export class SubscriptionViewer implements UpdatableSettingsViewer {
 							: !this._isCancelled
 							? m(IconButton, {
 									title: "subscription_label",
-									click: () => this.onSubscriptionClick(),
+									click: async () => await this.onSubscriptionClick(),
```

Comment:
please don't make it async and don't await there, just fire and forget.

Also it doesn't need to be async, looks like a leftover